### ISSUE TITLE
Have mpv as default player in default templates

### ISSUE THREAD

**Issue by emanruse** [QubesOS/qubes-issues#8696]
### The problem you're addressing (if any)

Improving video performance in the non-GPU accelerated environment of Qubes OS.

### The solution you'd like

Have `mpv` installed in default templates, as it seems to give best performance with parameters:

```
$ cat /etc/mpv/mpv.conf 
vo=x11
profile=sw-fast
```

This seems to work much better than any other options I have tried (VLC, mplayer, playing a video file in Firefox-based browser). The default player in Fedora templates (parole) doesn't even have the necessary codecs to play a video downloaded from YouTube, so it can be removed from the template (perhaps).

### The value to a user, and who that user might be

The suggested configuration allows a full screen video player (tested on 2K and 4K screens) with excellent performance when playing a 1080p video, even at speeds of x4.

Tested in a qube based on clean Debian 12 minimal template with additionally installed packages:

```
apt-get install --no-install-recommends mpv pulseaudio-qubes
```

Discussion:

https://forum.qubes-os.org/t/improving-video-playback-speed/21906/

**Comment by DemiMarie** [comment]
> I don't know whether [this](https://www.qubes-os.org/faq/#could-you-please-make-my-preference-the-default) applies here. I'll leave it up to the devs to decide.

I don’t think it does apply.  Fedora has other constraints, such as integration with GNOME, that do not apply here, and other players are likely optimized for an environment where GPU acceleration is available.  A player that only supported software decoding would be terrible for most people, but fantastic for Qubes OS.

If ITL needs to be concerned with software patents, then may be best to not ship any media player at all, but rather to allow users to download a flatpak that bundles its codecs.
